Powerup gui

dalderliesten opened the pull request:
-Implemented updating of the GUI label for the power-ups.
-Fixed associated javadoc.

Commit:
Make Default Constant

Made the default NONE text for a lack of a power-up a constant.

<organization

Commit:
Merge branch 'release' into powerup-gui

Commit:
Fix Main Javadoc

Fixed the javadoc to match the general agreed style for the project.

<organization

Commit:
Add TODO

Added todo flag for work to continue at a later stage.

<organization

Commit:
Implement Correct PowerUpPickUp

Removed the todo and added the correct current object for the returned
powerup arraylist.

<feature, organization

Commit:
Implement Chili Label

Added the Chili speed label and reset to the GUI.

<feature

Commit:
Implement taco label

Added support for the taco invincibility label to show in the GUI when
picked-up.

<feature

Commit:
Tweak Chili Label

Tweaked the Chili label to correctly go back to the NONE state.  Before,
it would say none too late.

<bug

Commit:
Fix Taco Label

Fixed the taco label, which would go back to NONE a second or two too
late.

<bug

NielsWarnars commented:
Label is still present if a player dies. If you have time you could consider changing that.
<review

dalderliesten commented:
Good catch, fixing now.

<review

Commit:
Fix Death Reset

Fixed an issue that caused the power-up label to not update back to NONE
after the death of all players.

<bug
